altair/vegalite/v2/examples/line_percent.py

```diff
@@ -0,0 +1,23 @@
+"""
+Line Chart with Percent axis
+-----------------------
+This example shows how to set an axis as a percent.
+"""
+
+import altair as alt
+from vega_datasets import data
+
+source = data.jobs()
+
+source.year = source.year.astype(str)
+
+welders = source[source.job == 'Welder']
+    
+chart = alt.Chart(welders).mark_line().encode(
+    x=alt.X('year', timeUnit='year'),
+    y=alt.Y('perc', axis=alt.Axis(format='%')),
+    color='sex',
+    detail='job'
```

Comment:
This line should be removed – since the data contains only one job, it doesn't do anything.